nemo_skills/dataset/rasb-26h1/prepare.py

```
@@ -0,0 +1,208 @@
+        for input_file in input_files:
+            entry = create_entry(env_path, input_file, metadata, data_dir)
+            entries.append(entry)
+            input_count += 1
+
+    # Write output
+    with open(output_file, "w", encoding="utf-8") as f:
+        for entry in entries:
+            f.write(json.dumps(entry) + "\n")
+
+    # Summary
+    print(f"\nRASB 26H1 Preparation Summary")
```

Ruff (0.15.11)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)